roles/zitadel/templates/values.yaml.j2

```diff
@@ -0,0 +1,61 @@
+---
+zitadel:
+  masterkey: {{ zitadel_master_key}}
+  secretConfig:
+    Database:
+      Postgres:
+        User:
+          Password: {{ zitadel_db_password }}
+        Admin:
+          Password: {{ zitadel_db_password }}
+  configmapConfig:
+    FirstInstance:
+      InstanceName: MetalStack
+      Org:
+        Name: MetalStack
+        Human:
+          Username: admin
+          FirstName: Metal
+          LastName: Stack
+          NickName: 
+          DisplayName: MetalStack Admin
+          Password: {{ zitadel_admin_password }}
+          PasswordChangeRequired: false
+    ExternalDomain: {{ zitadel_domain }}
+    ExternalPort: 80
+    TLS:
+      Enabled: false
+    Database:
+      Postgres:
+        Host: {{ zitadel_db_address }}
+        Port: 5432
+        Database: zitadel
+        MaxOpenConns: 20
+        MaxIdleConns: 10
+        MaxConnLifetime: 30m
+        MaxConnIdleTime: 5m
+        User:
+          Username: postgres
+          SSL:
+            Mode: disable
+        Admin:
+          Username: postgres
+          SSL:
+            Mode: disable
+ingress:
+  enabled: true
+  className: nginx
+env:
+  - name: ZITADEL_API_URL
+    value: {{ zitadel_ingress_dns }}
+image:
+  tag: v4.6.4
+login:
+  image:
+    tag: v4.6.4
```

Comment:
Probably this should not be overwritten as on a chart update this prevents an update?